server/src/controllers/treesOrganization.controller.ts

```diff
@@ -0,0 +1,185 @@
+import {
+  Count,
+  CountSchema,
+  Filter,
+  repository,
+  Where,
+} from '@loopback/repository';
+import {
+  post,
+  param,
+  get,
+  getFilterSchemaFor,
+  getWhereSchemaFor,
+  patch,
+  put,
+  del,
+  requestBody,
+  HttpErrors,
+} from '@loopback/rest';
+import {Trees} from '../models';
+import {TreesRepository} from '../repositories';
+//import expect from "../utils/expect.js";
+const expect : any = require("../utils/expect.js").default;
+
+export class TreesOrganizationController {
+  constructor(
+    @repository(TreesRepository)
+    public treesRepository : TreesRepository,
+  ) {}
+
+  @get('/organization/{organizationId}/trees/count', {
+    responses: {
+      '200': {
+        description: 'Trees model count',
+        content: {'application/json': {schema: CountSchema}},
+      },
+    },
+  })
+  async count(
+    @param.path.number('organizationId') organizationId: number,
+    @param.query.object('where', getWhereSchemaFor(Trees)) where?: Where<Trees>,
+  ): Promise<Count> {
+    const entityIds = await this.treesRepository.getEntityIdsByOrganizationId(organizationId);
+    const planterIds = await this.treesRepository.getPlanterIdsByOrganizationId(organizationId);
+    where = {
+      ...where,
+      or: [
```

Comment:
Good.   We are making one change in the database that will change this soon, but we need this OR for now.  When Mark is finished though we will have a database trigger to set planting_organization_id when trees are created based on the planter's organization, so we don't have an issue if the planter starts planting for a different organization.